### PR TITLE
Tvastr ops patch 1

### PR DESCRIPTION
**Pull request by Tvastr-ops**

### CODE REVIEW

File: .github/workflows/aws.yml [hunk 1]
```diff
@@ -0,0 +1,82 @@
...
+        container-name: sample-app
+        image: ${{ steps.build-image.outputs.image }}
+
+    - name: Deploy Amazon ECS task definition
+      uses: aws-actions/amazon-ecs-deploy-task-definition@v1
+      with:
+        task-definition: ${{ steps.task-def.outputs.task-definition }}
+        service: sample-app-service
+        cluster: default
+        wait-for-service-stability: true
```
Comment (karlhorky) [inline .github/workflows/aws.yml:82]: @Tvastr-ops what is this file for?

File: README.md [hunk 2]
```diff
@@ -7,6 +7,10 @@ This list aims to be a curated set of high quality educational resources. The av
 
 ## Beginner
 
+#### [CS50's Introduction to Computer Science](https://www.edx.org/learn/computer-science/harvard-university-cs50-s-introduction-to-computer-science/)
+**Free** A beginner friendly Computer Science & Programming course straight from Havards available on Edx.
+*(Scratch, C, Python, SQL, JavaScript, CSS, HTML)*
+
```
Comment (karlhorky) [inline README.md:13]: we aleady list edX further down in the list, maybe this doesn't make sense?

File: .github/workflows/aws.yml (hunk 1 above)
Comment (Tvastr-ops) [reply .github/workflows/aws.yml:82]: It just adds CS50 course in the readme document

Comment (karlhorky) [reply .github/workflows/aws.yml:82]: no, this file is not for that. it is a GitHub Actions workflow

I'm asking what purpose you added it for

Comment (Tvastr-ops) [reply .github/workflows/aws.yml:82]: It automatically got added. I didn't add it

Comment (karlhorky) [reply .github/workflows/aws.yml:82]: ok understood. however, it is now a part of your PR, and apparently it should not be, so you'll need to remove it

File: README.md (hunk 2 above)
Comment (Tvastr-ops) [reply README.md:13]: I see thank you. I didn't noticed. I had forked this repo a few years ago. And when I did workflow got created. I'm not used to GitHub so apologize for inconvenience I caused you.

Comment (karlhorky) [reply README.md:13]: no worries, no inconvenience :) 

happy to guide and help out while you get accustomed to the PR workflow

but let's close this one for now, and if you see any other improvements, you can try to open a new PR

**Comment by karlhorky** [comment]
Closing as discussed above

File: .github/workflows/aws.yml (hunk 1 above)
Comment (Tvastr-ops) [reply .github/workflows/aws.yml:82]: Ty I removed the workflow directory. But since Edx is already there I will just close the PR.

**Pull request closed without merging**